apps/docs/components/common/newsletter-signup.tsx

```diff
@@ -33,7 +33,12 @@ export function NewsletterSignup({
 			setFormState('loading')
 			try {
 				const _email = new FormData(e.currentTarget)?.get('email') as string
-				const { error } = await submitNewsletterSignup(_email)
+				const cookie = document.cookie
```

Comment:
I guess if the users allowed cookies well get it otherwise it should not be there.

Reply:
just a nit here: we should call the variable hubspotCookie to. make it clear this isn't the whole cookie.

Reply:
Done 👍